src/services/authorizationService.js

```diff
@@ -1,5 +1,6 @@
 import httpService from './httpService';
 import { ApiEndpoint } from '../utils/URL';
 
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
You would need to fix this error, instead of using `eslint-disable-next-line` to disable it.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/d9efa53e-b5ed-428d-b128-a9132a8c8422)